### ISSUE TITLE
Add visually hidden text to share links component

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk_publishing_components#1064]
We've [implemented a workaround within Collections](https://github.com/alphagov/collections/blob/2fd99c9f1456301cec48cbe78f35fc0a4e34aa8e/app/views/brexit_landing_page/_common.html.erb#L35) for this, but it should be an option in the component itself.
